src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -288,6 +288,16 @@ private static void PopulateAllSystemTimeZones(CachedData cachedData)
         {
             Debug.Assert(Monitor.IsEntered(cachedData));
 
+            cachedData._systemTimeZones ??= new Dictionary<string, TimeZoneInfo>(StringComparer.OrdinalIgnoreCase)
+            {
+                { UtcId, s_utcTimeZone }
+            };
```

Comment:
should this get moved inside the Invariant block?

Could the UTC get enumerated again in GetTimeZoneIds()?

Reply:
`??=` will assign it only once